rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -323,6 +323,32 @@ public void getLockingCap() {
         verify(provider, times(1)).setLockingCap(constants.getInitialLockingCap());
     }
 
+    @Test
+    public void getActivePowpegRedeemScriptBeforeFork() {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP293)).thenReturn(false);
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+                .withBridgeConstants(bridgeConstants)
+                .withActivations(activations)
+                .build();
+
+        assertEquals(0, bridgeSupport.getActivePowpegRedeemScript().getProgram().length);
+    }
+
+    @Test
+    public void getActivePowpegRedeemScriptAfterFork() {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP293)).thenReturn(true);
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+                .withBridgeConstants(bridgeConstants)
+                .withActivations(activations)
+                .build();
+
+        assertEquals(bridgeSupport.getActiveFederation().getRedeemScript(), bridgeSupport.getActivePowpegRedeemScript());
```

Comment:
Maybe it's better if we assert against the actual genesis federation redeem script here. This way it's basically asserting the same result calling the same method